### ISSUE TITLE
Power Connections cannot be retrieved by the ID of both connected devices

### ISSUE THREAD

**Issue by mshems** [netbox-community/netbox#5862]
### Environment
* Python version: 3.8.5
* NetBox version: v2.10.4
* Netbox-Docker version: 1.0.2

### Steps to Reproduce
1. Create Device A (will have id# 1) via UI
2. Add power outlet to Device A via UI
3. Create Device B (will have id#2) via UI
4. Add power port to Device B via UI
5. Connect outlet on Device A to port on Device B
6. Make request to ".../api/dcim/power-connections/?device_id=1"
7. Make request to ".../api/dcim/power-connections/?device_id=2"

 (requests were made via the Swagger API browser linked at the bottom of the UI)

### Expected Behavior
The two requests should return the same data -- one connection between device A and device B

### Observed Behavior
The first request returns no results
```
{
  "count": 0,
  "next": null,
  "previous": null,
  "results": []
}
```
### Additional Info
In another Netbox instance running v2.9.4, I was able to successfully retrieve power connections using the device ID of either connected device. I can no longer do this in Netbox v2.10+

**Comment by mshems** [comment]
This looks to be related to/an extension of this Issue: https://github.com/netbox-community/netbox/issues/5627

**Comment by jeremystretch** [comment]
Yeah, I'd say this is very likely covered by any fix for #5627. Let's close this out as a suspected duplicate, but feel free to re-open if the problem persists after #5627 has been addressed.

**Comment by mshems** [comment]
Sounds good, thanks for the quick response!
